mee/spi.h

```diff
@@ -0,0 +1,47 @@
+/* Copyright 2018 SiFive, Inc */
+/* SPDX-License-Identifier: Apache-2.0 */
+
+#ifndef MEE__SPI_H
+#define MEE__SPI_H
+
+struct mee_spi;
+
+struct mee_spi_config {
+    enum {
+        SPI_SINGLE,
```

Comment:
These should be within the MEE namespace.

Reply:
You mean prepend `MEE_` to these, right?